client/src/components/ContentNavigator/index.ts

```diff
@@ -4,21 +4,30 @@ import {
   ConfigurationChangeEvent,
   Disposable,
   ExtensionContext,
+  ProgressLocation,
   Uri,
   commands,
   l10n,
   window,
   workspace,
 } from "vscode";
 
+import { readFileSync } from "fs";
```

Comment:
It would be great if we can use `vscode.workspace.fs` so that virtual file system can also be supported.

Reply:
Done https://github.com/sassoftware/vscode-sas-extension/pull/552/commits/a92335600c9a59552ef5b4defccf362448337f24